### DISCUSSION TITLE
v2的更新进度

### DISCUSSION THREAD

**Discussion by Leepuding** [Wox-launcher/Wox#4065] [Q&A]
First time I came across a tool like wox thought it was great!I like it very much, I'm a windows system and would like to know when I can use the full-fledged v2!

**Comment by qianlifeng** [comment]
The Windows version is mainly missing the Everything plugin compared to v1, as well as some minor bugs that do not affect usage. It is generally usable at present.

**Reply by Leepuding** [reply]
Well, I'll try using it.

**Reply by Leepuding** [reply]
I am a Chinese person, the main application system language is mainly in Chinese, v2 will update the Chinese language, I do not see the option to switch the language in the settings.

**Reply by qianlifeng** [reply]
Partial multilingual support has been added in the latest version.
